### PR TITLE
set restaurant public key

### PR DESCRIPTION
**Pull request by lucasferraro**
We are sending the public key of the market place owner and the test in production has failed. 
I guess because we have to send the public key of the restaurant instead.

### CODE REVIEW

**Comment by lucasferraro** [comment]
I convert this PR to draft beacuse is better if we first merge #3027.
After merge #3027 I need to make some extra changes on this PR to support MercadoPago in forms.

**Comment by alexsegura** [comment]
@lucasferraro #3027 is now merged 😉

**Comment by lucasferraro** [comment]
@alexsegura Now it's ready :)
When this PR is merged we need it deployed on TRU instance so we can test Mercadopago in prod.
Thanks!

File: js/app/payment/mercadopago.js [hunk 1]
```diff
@@ -123,11 +123,10 @@ export default ({ onChange }) => ({
     this.form = form
     const { country, publishableKey } = this.config.gatewayConfig
 
-    /*
-     * When this flow is tested in a production environment we have to check if the Public Key to send
-     * has to be from the Marketplace owner or from the Seller/Restaurant.
-     * Now we are using the Public Key of the Marketplace owner (Coop).
-     */
+    if (!publishableKey) {
+      throw new Error('You need a Public Key for Mercadopago')
```
Comment (alexsegura) [inline js/app/payment/mercadopago.js:127]: This means it won't work for delivery form, right?

Comment (alexsegura) [reply js/app/payment/mercadopago.js:127]: Ok my bad. Perfect 🙂

**Pull request merged**